sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2_0/aio/operations/_form_recognizer_client_operations.py

```diff
@@ -286,6 +286,8 @@ async def begin_analyze_with_custom_model(
         :type include_text_details: bool
         :param file_stream: .json, .pdf, .jpg, .png or .tiff type file stream.
         :type file_stream: IO or ~azure.ai.formrecognizer.models.SourcePath
+        :keyword str content_type: Media type of the body sent to the API. Default value is "application/json".
```

Comment:
should we change Media for Content? just so it matches the name of the variable too?

Reply:
This is in the generated code, but I can update it elsewhere